sentry/src/lib.rs

```diff
@@ -123,5 +165,5 @@ pub fn bad_request(error: Box<dyn std::error::Error>) -> Response<Body> {
 pub struct Session {
     pub era: i64,
     pub uid: String,
-    pub ip: String,
+    pub ip: Option<String>,
```

Comment:
Accordingly to the JS impl [reference](https://github.com/AdExNetwork/adex-validator/blob/05-stable/routes/channel.js#L258)